apps/admin/frontend/package.json

```diff
@@ -125,10 +125,9 @@
   },
   "devDependencies": {
     "@jest/types": "^29.6.1",
-    "@testing-library/jest-dom": "^5.16.4",
-    "@testing-library/react": "^12.1.5",
-    "@testing-library/react-hooks": "^8.0.1",
-    "@testing-library/user-event": "^12.6.0",
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^14.0.0",
+    "@testing-library/user-event": "^13.5.0",
```

Comment:
We have to upgrade `@testing-library/react` two major versions to v14 for React 18 compatibility, and it's not backward compatible so it had to happen at the same time.

`@testing-library/user-event` only needs to go to v13, though, which is good because v14 introduces a ton of breaking changes.